tests/config_test.go

```diff
@@ -58,14 +60,25 @@ func TestInferConfigWrap(t *testing.T) {
 				baseConfigureWasCalled = true
 				return nil
 			},
+			Check: func(ctx context.Context, _ p.CheckRequest) (p.CheckResponse, error) {
+				infer.GetConfig[*testConfig](ctx) // Will panic if config is missing
```

Comment:
It would be clearer for the next person to have an assertion failure instead of a panic.